Fix gio/gcancellable.c build when thread support is disabled

Changes:
	Updated for 2.36.1

Signed-off-by: "Samuel Martin" <s.martin49@gmail.com>
Signed-off-by: "Spenser Gilliland" <spenser@gillilanding.com>
---
--- libglib2-2.36.1.orig/gio/gcancellable.c	2013-03-13 08:39:09.000000000 -0500
+++ libglib2-2.36.1/gio/gcancellable.c	2013-05-26 10:49:32.366019891 -0500
@@ -267,7 +267,9 @@
   while (priv->cancelled_running)
     {
       priv->cancelled_running_waiting = TRUE;
+#if defined(G_THREADS_ENABLED)
       g_cond_wait (&cancellable_cond, &cancellable_mutex);
+#endif
     }
 
   if (priv->cancelled)
@@ -620,7 +622,9 @@
   while (priv->cancelled_running)
     {
       priv->cancelled_running_waiting = TRUE;
+#if defined(G_THREADS_ENABLED)
       g_cond_wait (&cancellable_cond, &cancellable_mutex);
+#endif
     }
 
   g_signal_handler_disconnect (cancellable, handler_id);
